varats/varats/projects/c_projects/gzip.py

```diff
@@ -91,18 +91,23 @@ def compile(self) -> None:
         """Compile the project."""
         gzip_version_source = local.path(self.source_of_primary)
 
+        # Build binaries in separate dir because executing the binary with path 'gzip' will execute '/usr/bin/gzip' independent of the current working directory.
+        mkdir("-p", gzip_version_source / "build")
+
         self.cflags += [
             "-Wno-error=string-plus-int", "-Wno-error=shift-negative-value",
             "-Wno-string-plus-int", "-Wno-shift-negative-value"
         ]
 
-        clang = bb.compiler.cc(self)
         with local.cwd(gzip_version_source):
-            with local.env(CC=str(clang)):
-                bb.watch(local["./bootstrap"])()
-                bb.watch(local["./configure"])()
+            bb.watch(local["./bootstrap"])()
+
+        clang = bb.compiler.cc(self)
+        with local.cwd(gzip_version_source / "build"), local.env(CC=str(clang)):
```

Comment:
```suggestion
        c_compiler = bb.compiler.cc(self)
        with local.cwd(gzip_version_source / "build"), local.env(CC=str(c_compiler)):
```
When we already hear, fixing this we can clean up the old "bad" name :laughing: